Building on Fedora 29 results in errors

Several issues reported before the build failed:

```
[ 30%] Building CXX object core/CMakeFiles/core.dir/producer/stage.cpp.o
[ 31%] Building CXX object core/CMakeFiles/core.dir/StdAfx.cpp.o
[ 32%] Building CXX object core/CMakeFiles/core.dir/video_channel.cpp.o
[ 33%] Building CXX object core/CMakeFiles/core.dir/video_format.cpp.o
/home/gbraad/Downloads/server/src/core/video_channel.cpp: In lambda function:
/home/gbraad/Downloads/server/src/core/video_channel.cpp:116:25: error: ‘nb_samples’ was not declared in this scope
                         nb_samples = audio_cadence_.front();
                         ^~~~~~~~~~
/home/gbraad/Downloads/server/src/core/video_channel.cpp:116:25: note: suggested alternative: ‘glIsSampler’
                         nb_samples = audio_cadence_.front();
                         ^~~~~~~~~~
                         glIsSampler
/home/gbraad/Downloads/server/src/core/video_channel.cpp:116:38: error: ‘audio_cadence_’ was not declared in this scope
                         nb_samples = audio_cadence_.front();
                                      ^~~~~~~~~~~~~~
/home/gbraad/Downloads/server/src/core/video_channel.cpp:116:38: note: suggested alternative: ‘find_audio_cadence’
                         nb_samples = audio_cadence_.front();
                                      ^~~~~~~~~~~~~~
                                      find_audio_cadence
/home/gbraad/Downloads/server/src/core/video_channel.cpp:119:40: error: expected ‘;’ before ‘auto’
                     frame_counter_ += 1
                                        ^
                                        ;
/home/gbraad/Downloads/server/src/core/video_channel.cpp:121:21:
                     auto nb_samples = format_desc.audio_cadence[frame_counter_ % format_desc.audio_cadence.size()];
                     ~~~~                
/home/gbraad/Downloads/server/src/core/video_channel.cpp:143:70: error: ‘nb_samples’ was not declared in this scope
                     auto          stage_frames = stage_(format_desc, nb_samples, background_routes);
                                                                      ^~~~~~~~~~
/home/gbraad/Downloads/server/src/core/video_channel.cpp:143:70: note: suggested alternative: ‘glIsSampler’
                     auto          stage_frames = stage_(format_desc, nb_samples, background_routes);
                                                                      ^~~~~~~~~~
                                                                      glIsSampler
/home/gbraad/Downloads/server/src/core/video_channel.cpp:150:36: error: unable to deduce ‘auto&&’ from ‘stage_frames’
                     for (auto& p : stage_frames) {
                                    ^~~~~~~~~~~~
make[2]: *** [core/CMakeFiles/core.dir/build.make:389: core/CMakeFiles/core.dir/video_channel.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:258: core/CMakeFiles/core.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
```

I used mostly development libraries as provided by Fedoras repositories, with the exception of boost, and cef as these were extracted from the Docker images.

the following errors remain:

```
[ 32%] Building CXX object core/CMakeFiles/core.dir/video_channel.cpp.o
[ 33%] Building CXX object core/CMakeFiles/core.dir/video_format.cpp.o
/home/gbraad/Downloads/server/src/core/video_channel.cpp: In lambda function:
/home/gbraad/Downloads/server/src/core/video_channel.cpp:118:40: error: expected ‘;’ before ‘auto’
                     frame_counter_ += 1
                                        ^
                                        ;
/home/gbraad/Downloads/server/src/core/video_channel.cpp:120:21:
                     auto nb_samples = format_desc.audio_cadence[frame_counter_ % format_desc.audio_cadence.size()];
                     ~~~~                
/home/gbraad/Downloads/server/src/core/video_channel.cpp:142:70: error: ‘nb_samples’ was not declared in this scope
                     auto          stage_frames = stage_(format_desc, nb_samples, background_routes);
                                                                      ^~~~~~~~~~
/home/gbraad/Downloads/server/src/core/video_channel.cpp:142:70: note: suggested alternative: ‘glIsSampler’
                     auto          stage_frames = stage_(format_desc, nb_samples, background_routes);
                                                                      ^~~~~~~~~~
                                                                      glIsSampler
/home/gbraad/Downloads/server/src/core/video_channel.cpp:149:36: error: unable to deduce ‘auto&&’ from ‘stage_frames’
                     for (auto& p : stage_frames) {
                                    ^~~~~~~~~~~~
make[2]: *** [core/CMakeFiles/core.dir/build.make:389: core/CMakeFiles/core.dir/video_channel.cpp.o] Error 1
```

--- a/src/core/video_channel.cpp
+++ b/src/core/video_channel.cpp
@@ -115,7 +115,7 @@ struct video_channel::impl final
                         format_desc = format_desc_;
                     }
 
-                    frame_counter_ += 1
+                    frame_counter_ += 1;
 
                     auto nb_samples = format_desc.audio_cadence[frame_counter_ % format_desc.audio_cadence.size()];
 

